### ISSUE TITLE
gatsby-cli 2.12.105: Cannot find module 'gatsby-cli/src/structured-errors/error-map' or its corresponding type declarations.

### ISSUE THREAD

**Issue by kevcenteno** [gatsbyjs/gatsby#27339]
## Description

`gatsby develop` results in a typescript error

### Steps to reproduce

1. Set up a basic gatsby project with typescript
2. Make sure you have Gatsby-cli 2.12.105 installed
3. Run `yarn develop`

### Expected result

Development bundle building succeeds

### Actual result

Development bundle building fails:

```
ERROR in /home/gatsby-project/node_modules/gatsby-cli/lib/reporter/redux/types.d.ts(3,31):
3:31 Cannot find module 'gatsby-cli/src/structured-errors/error-map' or its corresponding type declarations.
    1 | import { Actions, ActivityStatuses, ActivityTypes } from "../constants";
    2 | import { IStructuredError } from "../../structured-errors/types";
  > 3 | import { ErrorCategory } from "gatsby-cli/src/structured-errors/error-map";
      |                               ^
    4 | export interface IGatsbyCLIState {
    5 |     messages: Array<ILog>;
    6 |     activities: {
failed Building development bundle - 17.058s
```

### Environment
```
  System:
    OS: Linux 5.4 Ubuntu 20.04.1 LTS (Focal Fossa)
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 10.16.0 - /tmp/yarn--1602164833852-0.12649155661585176/node
    Yarn: 1.21.1 - /tmp/yarn--1602164833852-0.12649155661585176/yarn
    npm: 6.9.0 - ~/.nvm/versions/node/v10.16.0/bin/npm
  Languages:
    Python: 2.7.18 - /usr/bin/python
  Browsers:
    Chrome: 86.0.4240.75
  npmPackages:
    gatsby: ^2.20.15 => 2.24.71 
    gatsby-image: ^2.3.2 => 2.4.21 
    gatsby-plugin-csp: ^1.1.3 => 1.1.3 
    gatsby-plugin-manifest: ^2.3.3 => 2.4.34 
    gatsby-plugin-mdx: ^1.2.26 => 1.2.46 
    gatsby-plugin-offline: ^3.1.2 => 3.2.31 
    gatsby-plugin-react-helmet: ^3.2.1 => 3.3.13 
    gatsby-plugin-react-svg: ^3.0.0 => 3.0.0 
    gatsby-plugin-sass: ^2.2.1 => 2.3.16 
    gatsby-plugin-sharp: ^2.5.4 => 2.6.39 
    gatsby-plugin-sri: ^1.1.0 => 1.1.0 
    gatsby-plugin-typescript: ^2.3.1 => 2.4.21 
    gatsby-plugin-typescript-checker: ^1.1.1 => 1.1.1 
    gatsby-plugin-webpack-bundle-analyzer: ^1.0.5 => 1.0.5 
    gatsby-source-filesystem: ^2.2.2 => 2.3.34 
    gatsby-transformer-sharp: ^2.4.4 => 2.5.17 
```

**Comment by kevcenteno** [comment]
This looks like the offending commit: https://github.com/gatsbyjs/gatsby/commit/726d56c975a321455fcd234e4be656d977ccfc02

**Comment by LekoArts** [comment]
Sorry for the troubles, this was caused by https://github.com/gatsbyjs/gatsby/pull/27176 and will be fixed by #27340 after a new publish

**Comment by kevcenteno** [comment]
> Sorry for the troubles, this was caused by #27176 and will be fixed by #27340 after a new publish

No worries.  I appreciate the quick turnaround!

**Comment by LekoArts** [comment]
```
Successfully published:
 - gatsby-admin@0.1.161
 - gatsby-cli@2.12.106
 - gatsby@2.24.72
```
